### ISSUE TITLE
Any plans to support CT/SCT?

### ISSUE THREAD

**Issue by OdinVex** [chris2511/xca#323]
Any plans to support Certificate Transparency/Signed Certificate Timestamp?

**Comment by temojohnson** [comment]
Finally, some else who same question that I have! I hope they do support SCT/CT Logs.

**Comment by chris2511** [comment]
No, currently there are no plans to support CT/SCT in the near future.
(To be honest: I'm not very familiar with this topic, yet)

However, pull requests are always welcome.

**Comment by OdinVex** [comment]
> No, currently there are no plans to support CT/SCT in the near future. (To be honest: I'm not very familiar with this topic, yet)
> 
> However, pull requests are always welcome.

I was informed Chrome will be requiring CT/SCT in order for EV/DV/etc. to work, soon, too. I don't do any Qt-development or I'd pitch in.

**Comment by LeviMarvin** [comment]
Please support CT / SCT, it will be mandatory need in the future.

**Comment by itsKV** [comment]
+1 for ct/sct support.
